app/scripts/models/auth_brokers/base.js

```diff
@@ -193,8 +224,12 @@ define([
      * @param {object} account
      * @return {promise}
      */
-    afterCompleteSignUp: function (/* account */) {
-      return p(this.getBehavior('afterCompleteSignUp'));
+    afterCompleteSignUp: function (account) {
+      var self = this;
+      return self.clearPersisted(account)
```

Comment:
In a soonish-future, I'd like to have the brokers manage `persist` as well, and move that responsibility [out of the views](https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/views/confirm.js#L99)